lib/readline.js

```diff
@@ -383,6 +398,8 @@ Interface.prototype._refreshLine = function() {
   }
 
   this.prevRows = cursorPos.rows;
+
+  searchHistory.call(this);
```

Comment:
This calls the searchHistory more often than it's necessary. Right now it would trigger this with each e.g. cursor move but we should only trigger the search if the input letters changed.